src/backend/distributed/transaction/backend_data.c

```diff
@@ -945,3 +960,68 @@ GetMyProcLocalTransactionId(void)
 {
 	return MyProc->lxid;
 }
+
+
+/*
+ * get_all_active_client_backend_count returns the active
+ * client backend count.
+ */
+Datum
```

Comment:
Should this UDF go to src/test or exposed to the users? I'm OK to expose to test things easier

Reply:
test seems preferable, we have too many UDFs

Reply:
moved to tests